### PR TITLE
Bump Azure integration

### PR DESCRIPTION
**Pull request by Shr3ps**

### CODE REVIEW

**Review by BzSpi — APPROVED** [review]

**Comment by xp-1000** [comment]
hello @Shr3ps, conflicts must be resolved

**Review by xp-1000 — CHANGES_REQUESTED** [review]
conflict to resolve

File: cloud/azure/versions.tf [hunk 1]
```diff
@@ -2,16 +2,16 @@ terraform {
...
+      version = "~> 3.1"
     }
     azuread = {
       source  = "hashicorp/azuread"
-      version = ">= 2"
+      version = "~> 2.0"
     }
   }
-  required_version = ">= 0.12.26"
+  required_version = ">= 0.13"
```
Comment (xp-1000) [inline cloud/azure/versions.tf:16]: why?

**Review by xp-1000 — CHANGES_REQUESTED** [review]

File: cloud/azure/versions.tf (hunk 1 above)
Comment (Shr3ps) [reply cloud/azure/versions.tf:16]: The goal is to align all the Azure Terraform ecosystem we have, it includes some changes in our claranet-tfwrapper which requires Terraform 0.13 (or 1.0 in some cases)

**Review by pdecat — APPROVED** [review]

**Pull request merged**